vercel.json

```diff
@@ -1091,7 +1061,7 @@
       "permanent": true
     },
     {
-      "source": "/next/tools/toolchains/flow-cli/:path*",
+      "source": "/next/tools/flow-cli/:path*",
```

Comment:
doesn't this mean that we lose an existing redirect?

Reply:
yes, we'll lose the redirect, but it was here for less than a week and it was for the next version